docs/api/ipc-renderer.md

```diff
@@ -143,6 +143,35 @@ Sends a message to a window with `webContentsId` via `channel`.
 Like `ipcRenderer.send` but the event will be sent to the `<webview>` element in
 the host page instead of the main process.
 
+### `ipcRenderer.postMessage(channel, message, [options])`
+
+* `channel` String
+* `message` any
+* `options` MessagePort[] (optional)
```

Comment:
Calling this options when it's an array of ports feels weird.  Either this should be an object of `{ ports[] }` or this should be called `ports`